extensions/GitExtension/FileExplorerGitIntegration/Models/RepositoryWrapper.cs

```diff
@@ -171,6 +172,26 @@ public string GetFileStatus(string relativePath)
         return statusString;
     }
 
+    private string GetOriginalPath(string relativePath)
+    {
+        _statusCache.Status.FileEntries.TryGetValue(relativePath, out var status);
+        if (status is null)
+        {
+            return relativePath;
+        }
+

```

Comment:
nit: may we add a comment here stating that if a file is renamed the old name is used as the change has not been committed. Thanks!